### PR TITLE
fix: increase tab max-width to handle bigger text

### PR DESCRIPTION
**Pull request by palashkaria**
## Description
Fixes: https://github.com/appsmithorg/appsmith/issues/22455
Fixed by increase the max-width from 138px to 180px.

#### PR fixes following issue(s): 
Fixes # (issue number): https://github.com/appsmithorg/appsmith/issues/22455  
- Bug fix (non-breaking change which fixes an issue)

Tested by adding a long text and verifying that it's visible

> Delete anything that is not relevant
- [x] Manual

>
>

#### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Test-plan-implementation#speedbreaker-features-to-consider-for-every-change) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans/_edit#areas-of-interest)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/src/widgets/TabsWidget/component/PageTabs.tsx [hunk 1]
```diff
@@ -47,11 +47,11 @@ const StyleTabText = styled.div<{
   color: ${(props) => getComplementaryGrayscaleColor(props.backgroundColor)};
   font-weight: normal;
   height: 32px;
-  max-width: 138px;
+  max-width: 180px;
   display: flex;
 
   & div {
-    max-width: 138px;
+    max-width: 180px;
```
Comment (vsvamsi1) [inline app/client/src/widgets/TabsWidget/component/PageTabs.tsx:54]: Should we make the font size relative, something like 12.5rem will come to approximately 200px for 25 characters?
cc @rajatagrawal

Comment (dilippitchika) [reply app/client/src/widgets/TabsWidget/component/PageTabs.tsx:54]: Yeah, this works as well. @palashkaria can you make the change and let us know?

Comment (rajatagrawal) [reply app/client/src/widgets/TabsWidget/component/PageTabs.tsx:54]: cool!

Comment (palashkaria) [reply app/client/src/widgets/TabsWidget/component/PageTabs.tsx]: This change is done @vsvamsi1 @rajatagrawal

**Review by vsvamsi1 — APPROVED** [review]

**Review by rajatagrawal — APPROVED** [review]

File: app/client/src/widgets/TabsWidget/component/PageTabs.tsx (hunk 1 above)
Comment (vsvamsi1) [reply app/client/src/widgets/TabsWidget/component/PageTabs.tsx]: Cool I have requested a qa verification on this @palashkaria.

**Comment by kamakshibhat-appsmith** [comment]
@palashkaria 
> Isn't this the case already? 
Well with our current UI , the padding looks consistent throughout. 

> This PR only changes the max-width, not spacing between tabs.
I think the padding fix is expected with width change, else the fix is gonna look half baked. [LOOM](https://www.loom.com/share/db9acec57a1a459d8b5ab734c727bf9c) for comparison

@dilippitchika @vsvamsi1 I don't think we need a separate PR to handle the padding case. But let me know if you feel otherwise . I will move the status accordingly

**Comment by dilippitchika** [comment]
@palashkaria i didn't expect the max-width to change the padding between tabs. Can you also make sure the padding is consistent?

**Comment by palashkaria** [comment]
This branch has been updated to fix the issue shared by @kamakshibhat-appsmith in the loom video. @dilippitchika @somangshu consider re-opening this PR? 

See [commit](https://github.com/palashkaria/appsmith/commit/65259ae316c410db074c7f40d5a6b711303e5fea)

**Comment by palashkaria** [comment]
Done, raised & tagged here @somangshu https://github.com/appsmithorg/appsmith/pull/24797

**Pull request closed without merging**